docs/UserGuide.adoc

```diff
@@ -60,6 +60,15 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 Format: `help`
 
+=== Changing the theme : `theme`
+
+Format: `theme t/THEME`
+
+Examples:
+
+* `theme t/light`
+* `theme t/dark`
+
 === Adding a person: `add`
```

Comment:
Is this feature still in? Remember to update your User Guide for every change to reduce the chances of any stale information misleading users.